integrators/embedded_explicit_generalized_runge_kutta_nyström_integrator.hpp

```diff
@@ -0,0 +1,158 @@
+﻿
+// The files containing the tree of of child classes of |Integrator| must be
+// included in the order of inheritance to avoid circular dependencies.  This
+// class will end up being reincluded as part of the implementation of its
+// parent.
+#ifndef PRINCIPIA_INTEGRATORS_INTEGRATORS_HPP_
+#include "integrators/integrators.hpp"
+#else
+#ifndef PRINCIPIA_INTEGRATORS_EMBEDDED_EXPLICIT_GENERALIZED_RUNGE_KUTTA_NYSTRÖM_INTEGRATOR_HPP_  // NOLINT(whitespace/line_length)
+#define PRINCIPIA_INTEGRATORS_EMBEDDED_EXPLICIT_GENERALIZED_RUNGE_KUTTA_NYSTRÖM_INTEGRATOR_HPP_  // NOLINT(whitespace/line_length)
+
+#include <functional>
+#include <vector>
+
+#include "base/not_null.hpp"
+#include "base/status.hpp"
+#include "numerics/fixed_arrays.hpp"
+#include "integrators/methods.hpp"
+#include "integrators/ordinary_differential_equations.hpp"
+#include "quantities/named_quantities.hpp"
+#include "serialization/integrators.pb.h"
+
+namespace principia {
+namespace integrators {
+namespace internal_embedded_explicit_generalized_runge_kutta_nyström_integrator {
+
+using base::not_null;
+using base::Status;
+using geometry::Instant;
+using numerics::FixedStrictlyLowerTriangularMatrix;
+using numerics::FixedVector;
+using quantities::Time;
+using quantities::Variation;
+
+// This class solves ordinary differential equations of the form
+//   q″ = f(q, q′, t)
+// using an embedded generalized Runge-Kutta-Nyström (RKNG) method.  We follow
```

Comment:
¯\\\_(ツ)\_/¯ 

Fine (1987) says “Runge-Kutta-Nyström Generalized (RKNG)”.
Sharp and Fine (1992) say “generalized Runge-Kutta-Nyström (RKNG)”.
Dormand (1996) says “The designation ‘RKNG’ is used because the shorter ‘RKN’ abbreviation is usually reserved for formulae applicable to the special equation (14.2)”, and does not expand the acronym.
Murua (1998) says “RKNG” without expanding the acronym.